librz/core/cmd_descs/cmd_debug.yaml

```diff
@@ -4,4 +4,47 @@ commands:
   - name: dc
     cname: debug_continue_oldhandler
     summary: Continue execution
-    type: RZ_CMD_DESC_TYPE_OLDINPUT
\ No newline at end of file
+    type: RZ_CMD_DESC_TYPE_OLDINPUT
+  - name: ds
+    summary: Debug step commands
+    subcommands:
+      - name: dsu
+        summary: Debug step until commands
+        subcommands:
+          - name: dsu
+            cname: cmd_debug_step_until
+            summary: Step until <address>
+            args:
+            - name: addr
+              type: RZ_CMD_ARG_TYPE_RZNUM
+              flags: RZ_CMD_ARG_FLAG_LAST
+          - name: dsui
+            cname: cmd_debug_step_until_instr
+            summary: Step until an instruction that matches <instr>
+            args:
+            - name: instr
+              type: RZ_CMD_ARG_TYPE_STRING
+          - name: dsuir
+            cname: cmd_debug_step_until_instr_regex
+            summary: Step until an instruction that matches <regex>
+            args:
+            - name: regex
+              type: RZ_CMD_ARG_TYPE_STRING
+          - name: dsuo
+            cname: cmd_debug_step_until_optype
+            summary: Step until an instruction matches one of the <optype>s
+            args:
+            - name: optype
+              type: RZ_CMD_ARG_TYPE_STRING
+          - name: dsue
+            cname: cmd_debug_step_until_esil
+            summary: Step until <esil> expression matches
+            args:
+            - name: esil
+              type: RZ_CMD_ARG_TYPE_STRING
+          - name: dsuf
+            cname: cmd_debug_step_until_flag
+            summary: Step until pc == <flag> matching name
+            args:
+            - name: flag
+              type: RZ_CMD_ARG_TYPE_STRING
```

Comment:
Please introduce a new type `RZ_CMD_ARG_TYPE_FLAG` for this. It will be useful once we start converting `f` commands as well and when i add autocompletion for flag names. If you use it now, it will be automatically autocompleted later ;)